packages/flutter_test/lib/src/window.dart

```diff
@@ -678,7 +678,8 @@ class TestFlutterView implements FlutterView {
   final TestDisplay _display;
 
   @override
-  Object get viewId => _view.viewId;
+  // TODO(goderbauer): Clean ignore and cast up after https://github.com/flutter/engine/pull/42493 landed.
```

Comment:
Unnecessary nit, sorry! 😅

```suggestion
  // TODO(goderbauer): Clean ignore and cast up after https://github.com/flutter/engine/pull/42493 lands.
```

Same comment for 2109. I don't feel strongly about this, feel free to skip.